scripts/save_tsdf_data.m

```diff
@@ -24,7 +24,7 @@
 
    
     % Run the python script save_tsdf_data.py and get the 3 outputs
-    [py_succ, ~] = pyrunfile("tsdf_python_interface\save_tsdf_data.py",["Success" "Return_list_metadata" "Return_list_data"], py_list_metadata=py_list_metadata, py_list_data=py_list_data, py_dir_path=py_dir_path, py_metadata_file_name=py_metadata_file_name);
+    [py_succ, ~] = pyrunfile("save_tsdf_data.py",["Success" "Return_list_metadata" "Return_list_data"], py_list_metadata=py_list_metadata, py_list_data=py_list_data, py_dir_path=py_dir_path, py_metadata_file_name=py_metadata_file_name);
```

Comment:
Just to double check, this will still work, even though the files are not in the same folder?

Reply:
Yes because in Matlab you basically add the entire wrapper directory to your path (using addpath(genpath(PATH/TO/TSDF4MATLAB) otherwise it cannot find the save_tsdf_data funciton itself. So this means that all other files in the directory are on the path already and you don't have the issue of linux/windows.